app/controllers/needs_controller.rb

```diff
@@ -10,8 +10,9 @@ class NeedsController < ApplicationController
   def index
     @params = params.permit(:user_id, :status, :category, :page, :order_dir, :order, :commit, :is_urgent)
     @users = User.all
```

Comment:
Thinking about future requirements - this User.all will probably need to be reviewed.  i.e. should anyone be able to assign to anyone else, or should it be a restricted set.

Reply:
Possibly true.  I don't think there's a requirement at the moment for who you can assign it to, but it does open the possibility of assigning to a user accidentally that you then can't see, e.g. a council service team assigning to a call centre user/team will no longer be able to see the need.

This can be solved currently via an admin user however